package/utils/wtmcu/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wtmcu
```

Comment:
Creating that package should be a separate commit before adding the actual device.